data.sql

```diff
@@ -42,4 +42,39 @@ UPDATE animals set owner_id = 3 WHERE name in ('Devimon','Plantmon');
 UPDATE animals set owner_id = 4 WHERE name in ('Charmander','Squirtle','Blossom');
 
 UPDATE animals set owner_id = 5 WHERE name in ('Angemon','Boarmon');
-       
\ No newline at end of file
+       
+--        Day 4
+
+INSERT INTO vets(id,name, age, date_of_graduation)
+	VALUES 	(1,'William Tatcher', 45, '2000-4-23'),
+			(2,'Maisy Smith', 26, '2019-2-17'),
+			(3,'Stephanie Mendez', 64, '1981-5-4'),
+			(4,'Jack Harkness', 38, '2008-1-8');
```

Comment:
- [ ] When your id becomes an autoincremented PRIMARY KEY, then it will be generated automatically without adding the id manually. Kindly see the example below and try to alter your insertions: 
```sql
INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('William Tatcher', 45, '2000-04-23');
```